ui/apps/platform/src/Containers/Violations/Details/Deployment/ContainerConfigurationDescriptionList.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement } from 'react';
+import { DescriptionList, Flex, FlexItem } from '@patternfly/react-core';
+
+import DescriptionListItem from 'Components/DescriptionListItem';
+import { Container } from 'types/deployment.proto';
+
+import ContainerImage from './ContainerImage';
+import ContainerResourcesDescriptionList from './ContainerResourcesDescriptionList';
+import ContainerSecretDescriptionList from './ContainerSecretDescriptionList';
+import ContainerVolumeDescriptionList from './ContainerVolumeDescriptionList';
+
+function MultilineDescription({ descArr }) {
+    return (
+        <Flex direction={{ default: 'column' }}>
+            {descArr.map((desc, idx) => (
+                // eslint-disable-next-line react/no-array-index-key
+                <div key={idx}>
+                    <FlexItem>{desc}</FlexItem>
+                </div>
+            ))}
+        </Flex>
+    );
+}
+
+type ContainerConfigurationDescriptionListProps = {
+    container: Container;
+};
+
+function ContainerConfigurationDescriptionList({
+    container,
+}: ContainerConfigurationDescriptionListProps): ReactElement {
+    const { resources, volumes, secrets, config, image } = container;
+    const { command, args } = config || {};
+    return (
+        <DescriptionList isCompact isHorizontal>
+            <ContainerImage image={image} />
+            {(command?.length > 0 || args?.length > 0) && (
+                <>
+                    {command.length > 0 && (
+                        <DescriptionListItem
+                            term="Commands"
+                            desc={<MultilineDescription descArr={command} />}
+                            aria-label="Commands"
+                        />
+                    )}
+                    {args?.length > 0 && (
+                        <DescriptionListItem
+                            term="Arguments"
+                            desc={<MultilineDescription descArr={args} />}
+                            aria-label="Arguments"
+                        />
+                    )}
+                </>
+            )}
+            {!!resources && (
+                <DescriptionListItem
+                    term="Resources"
+                    desc={
+                        resources ? (
+                            <ContainerResourcesDescriptionList resources={resources} />
+                        ) : (
+                            'None'
+                        )
```

Comment:
Nit: if this renders when `!!resources` is truthy, the "else" side of this ternary should never occur.